src/api/bookings/bookings.controller.js

```diff
@@ -12,3 +12,9 @@ export async function create(req, res) {
   const newBooking = await bookingsService.create({ bookingDataValidated });
   res.json(newBooking);
 }
+
+export async function logicDelete(req, res) {
```

Comment:
deberíais unificar nombres, no podéis llamar a la misma funcionalidad de distinta manera. Consulta con tus compañeros cómo llamar al borrado lógico